inst/WORDLIST

```diff
@@ -1,23 +1,13 @@
-ADSL
-ADTTE
-CDISC
-Dataname
 Forkers
 Github
 Hoffmann
-Kahn
 Pre
-README
-Repo
 Reproducibility
 UI
 cloneable
-dataname
-datanames
 funder
 iteratively
 pre
 repo
 reproducibility
 ui
```

Comment:
can we combine it with `"UI"` that we already have?

Reply:
https://github.com/insightsengineering/teal.data/pull/150/commits/e6cc0826b7dc5ee70f77bb5dad17a0e521c8b19b